DS-243 Update usage readme for jest (#40)
SonarSource/echoes-react: README.md

A React implementation of Echoes, Sonar's Design System.

## Installation

```bash
yarn add @sonarsource/echoes-react
```

## Usage

Make sure to setup the `IntlProvider` from `react-intl` at the top of your app. See [this page](https://formatjs.io/docs/react-intl/components/#intlprovider) for more information.

> 

Reviewer: NP: "root" rather than "top"?

Use components from the lib:

```ts
import { Checkbox } from '@sonarsource/echoes-react';
```

See available components and usage in storybook: https://echoes-react.netlify.app/

### Make it work in Jest

> The lib only provide es module bundle, so if you use Jest for your tests make sure your transform preprocessor goes through `echoes-react` to make it understood by Node.js, you can do that by adding an exception in your `transformIgnorePatterns`, for example:

Reviewer: ```suggestion
The lib only provides es module bundle. If you use Jest for your tests (or a similar library) make sure your transform preprocessor goes through `echoes-react` to make it runnable on Node.js. 
You can do that by adding an exception in your `transformIgnorePatterns`, for example:
```
